src/Theme/HookPreprocess.php

```diff
@@ -55,4 +58,59 @@ public static function block(array &$variables): void
             ->load($variables['elements']['#id']);
         $variables['block'] = new WrappedConfigEntity($block);
     }
+
+    /**
+     * @param array{
+     *   node: NodeInterface<string, FieldItemListInterface>,
+     *   teaser: bool,
+     *   url: string,
+     *   author_name?: MarkupInterface,
+     *   author_picture?: mixed[],
+     *   date?: MarkupInterface,
+     *   display_submitted?: bool,
+     *   label?: mixed[]
+     * } $variables
+     */
+    public static function node(array &$variables): void
+    {
+        $node = $variables['node'];
+        $variables['promote'] = $node->isPromoted();
+        $variables['sticky'] = $node->isSticky();
+        $variables['status'] = $node->isPublished();
+        $variables['preview'] = $node->in_preview ?? null;
+        $variables['name'] = $variables['author_name'] ?? '';
+        $variables['node_url'] = $variables['url'];
+        $variables['title'] = $variables['label'] ?? '';
+        $variables['submitted'] = '';
+        $variables['user_picture'] = '';
+        if (!empty($variables['display_submitted'])) {
+            if (isset($variables['date'])) {
+                $variables['submitted'] = t('Submitted by @username on @datetime', [
+                    '@username' => $variables['name'],
+                    '@datetime' => $variables['date'],
+                ]);
+            }
+            if (isset($variables['author_picture'])) {
+                $variables['user_picture'] = render($variables['author_picture']);
+            }
+        }
+        $variables['classes_array'][] = drupal_html_class('node-' . $node->bundle());
+        if ($variables['promote']) {
+            $variables['classes_array'][] = 'node-promoted';
+        }
+        if ($variables['sticky']) {
+            $variables['classes_array'][] = 'node-sticky';
+        }
+        if (!$variables['status']) {
+            $variables['classes_array'][] = 'node-unpublished';
+        }
+        if ($variables['teaser']) {
+            $variables['classes_array'][] = 'node-teaser';
+        }
+        if (isset($variables['preview'])) {
+            $variables['classes_array'][] = 'node-preview';
+        }
+        $variables['theme_hook_suggestions'][] = 'node__' . $node->bundle();
+        $variables['theme_hook_suggestions'][] = 'node__' . $node->id();
```

Comment:
See same questions from #132 if this is too late or if we need to provide a way to provide template suggestions like normal